playbooks/acs.yml

```diff
@@ -352,6 +349,26 @@
         repository_extra_war_downloads: "{{ acs_play_repository_extra_war_downloads }}"
         repository_extra_amp_downloads: "{{ acs_play_repository_extra_amp_downloads }}"
         repository_truststore_type: "{{ acs_play_default_truststore_type }}"
+    - name: Include HxInsight vars
+      ansible.builtin.include_vars:
+        file: ../vars/hxi.yml
```

Comment:
It's probably safe to use a `var_files` at the play level instead or plain `set_facts` tasks

Reply:
there is a conditional, which can be applied only to a task.

not sure ho set_facts can help here, hxi.yml contains configuration for hxinsight the user is meant to provide manually

just realized that I didn't write a single docs about this